cmd/openshift-install/gather.go

```diff
@@ -155,7 +155,11 @@ func logGatherBootstrap(bootstrap string, port int, masters []string, directory
 	if err := ssh.PullFileTo(client, fmt.Sprintf("/home/core/log-bundle-%s.tar.gz", gatherID), file); err != nil {
 		return errors.Wrap(err, "failed to pull log file from remote")
 	}
-	logrus.Infof("Bootstrap gather logs captured here %q", file)
+	path, err := filepath.Abs(file)
```

Comment:
a quick example, where we see how the bug was solved, in the PR description will be very nice.

Reply:
I've updated the PR description to reflect the fix.